### DISCUSSION TITLE
Release cycle schedule framework

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#3020] [General]
I guess this is one hard to estimate, what would be the best way to establish a "release cycle" schedule?
As we are going like superior beasts contributing a lot of stuff, would be nice to construct a flight plan for making release cycles, do's & don'ts.

Previously I've gone like approx every 14-21 days to push something out when you check https://github.com/kodadot/nft-gallery/releases 

Sometimes I've waited for to be "feature complete" stuff.

Now I'm checking that going amount of commits and significant user impact would be a great way to go.

Happy to hear your opinion, dropping to our Beta channel

![image](https://user-images.githubusercontent.com/5887929/168427183-e97d258e-9894-41e0-b261-29def3043402.png)

**Comment by vikiival** [comment]
I would target for 21 days + one week to write and release the article :)
